app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/ContinueInteractionViewModel.kt

```diff
@@ -65,7 +63,7 @@ class ContinueInteractionViewModel private constructor(
         hasPreviousButton,
         fragment as PreviousNavigationButtonListener,
         isSplitView,
-        writtenTranslationContext
+        writtenTranslationContext,
```

Comment:
```suggestion
        writtenTranslationContext
```

Small nit for consistency.

Reply:
done